src/client/Microsoft.Identity.Client.Broker/WamAdapters.cs

```diff
@@ -165,14 +165,23 @@ private enum ResponseStatus
             ILoggerAdapter logger)
         {
             logger.Verbose("[WamBroker] Validating Common Auth Parameters.");
-            ValidateAuthParams(authenticationRequestParameters, logger);
 
             var authParams = new NativeInterop.AuthParameters
                 (authenticationRequestParameters.AppConfig.ClientId,
                 authenticationRequestParameters.Authority.AuthorityInfo.CanonicalAuthority.ToString());
 
-            //scopes
-            authParams.RequestedScopes = string.Join(" ", authenticationRequestParameters.Scope);
+            //When no scopes are passed, add the default scopes
+            if (!ScopeHelper.HasNonMsalScopes(authenticationRequestParameters.Scope))
+            {
+                string defaultScopes = "email offline_access openid profile";
```

Comment:
We don't send email scope. Maybe create a `const string = "offline_access openid profile"` in OAuthConstants.
https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/a927f8c716677893c116333d6d8d2d12618f102b/src/client/Microsoft.Identity.Client/OAuth2/OAuthConstants.cs#L97-L99